### ISSUE TITLE
Update for newer IATI tools

### ISSUE THREAD

**Issue by matmaxgeds** [codeforIATI/iati-status#4]
We could consider updating this to do checks on:
- DSv2
- DSClassic
- ValidatorV2
- iaticonnect
- discuss 'archive'

**Comment by andylolz** [comment]
I’ve made a start on DSv2 (the activity and publisher counts, anyway).

**Comment by bill-anderson** [comment]
With datastore movements it would be good to get iati.cloud and datastore classic monitored asap. (If someone has time)

**Comment by andylolz** [comment]
Yep, agreed. It is a bit confusing at the moment, but Code for IATI managed products are monitored at:
https://status.codeforiati.org

…and secretariat managed products are monitored at:
https://iati-status.codeforiati.org

I think we at least need a link between the two things (easy), or we need to merge them onto a single subdomain (possibly a bit more involved.)
